bioscrape/types.pyx

```diff
@@ -717,10 +717,18 @@ def sympy_species_and_parameters(instring):
         index += 1
         nodes.extend(node.args)
 
-    names = [str(n) for n in nodes if type(n) == sympy.Symbol]
+    
+
+    #Old Way
+    #names = [str(n) for n in nodes if type(n) == sympy.Symbol]
+    #species_names = [s for s in names if (s[0] != '_' and s != 'volume' and s != 't')]
+    #param_names = [s[1:] for s in names if s[0] == '_']
 
-    species_names = [s for s in names if (s[0] != '_' and s != 'volume' and s != 't')]
-    param_names = [s[1:] for s in names if s[0] == '_']
+    #New Way
+    #remove leading "_" if there is one.
```

Comment:
we could also just not allow that? (leading underscores)

Reply:
I don't want to break all previous bioscrape models.